### PR TITLE
Ft riderequests tests 158787981

### PR DESCRIPTION
**Pull request by rethes**
#### What does this PR do?
Update ride tests

#### Description of Task to be completed?
Add error file
Add base tests
Utilise factory function across the app
Separate endpoint database and tests database

#### How should this be manually tested?
Using Postman

#### Any background context you want to provide?
Only the auth tests work, the ride tests fail because the endpoints are still being implemented
Utilising Postgres Database

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/158787981
https://www.pivotaltracker.com/story/show/158747503

### CODE REVIEW

File: tests/ridemyway_v1/test_base.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        ensure code is DRY[Do Not Repeat Yourself]
+        run before every single test
+        set values as instance[use self.] variable in order to access them whenever
+        """
+        self.app = create_app("testing")
+        self.app_context = self.app.app_context()
+        self.app_context.push()
+        # propagate the exceptions to the test client
+        self.tester = self.app.test_client(self)
+        create_all_tables()
```
Comment (Xerrex) [inline tests/ridemyway_v1/test_base.py]: This line might give a context error.
Try doing 
```
with self.app_context.push():
      create_all_tables()
```

File: myapp/test_database.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    """Closing the Database connection"""
+    try:
+        dbconn = g.pop('db', None)
+        # Close communication with the database
+        dbconn.close()
+    except:
+        pass
+
+
+def create_all_tables():
```
Comment (Xerrex) [inline myapp/test_database.py]: This function can be consolidated to 
query1 = "[statement]"
query2 = " [statement]"
query3 =" [statement]"
```
with conn.cursor() as cursor:
        cursor.execute(query1) 
        cursor.execute(query2) 
        cursor.execute(query3)   
        conn.commit()
        cursor.close()
        conn.close()
```

File: tests/ridemyway_v1/test_base.py (hunk 1 above)
Comment (rethes) [reply tests/ridemyway_v1/test_base.py]: The line does not give a context error after several tests, but I shall keep that in mind

File: myapp/test_database.py (hunk 2 above)
Comment (rethes) [reply myapp/test_database.py]: Change to be implemented

**Pull request merged**